package.json

```diff
@@ -52,11 +56,11 @@
     "semver": "4.3.2"
   },
   "dependencies": {
-    "@here/harp-geoutils": "^0.10.0",
-    "@here/harp-mapview": "^0.10.0",
+    "@here/harp-geoutils": "^0.11.0",
+    "@here/harp-mapview": "^0.11.0",
```

Comment:
`@here/harp-geoutils` should not be needed here because it is pulled by @here/harp-mapview
 also i believe harp-mapview should be  a peer/dev dependency

Reply:
> @here/harp-geoutils

@robertoraggi 
Hmm, good practice is to list all the deps that we explicitly depend on and we explicitly use @hare/harp-geoutils in this package, so i'll leave it heere ...

Both leaflet and harp deps moved to peerDepenies.